src/routes/notifications/notifications.service.ts

```diff
@@ -76,6 +76,27 @@ export class NotificationsService {
     }
   }
 
+  /**
+   * Sends a delete request for a chain, notification uuid and safe address.
+   * The uuid is expected to be managed by the client. Its value should be equal
+   * to the provided when the client called {@link registerDevice}.
```

Comment:
```suggestion
   * The uuid is expected to be managed by the client. Its value should be equal
   * to the one provided when the client called {@link registerDevice}.
```

Reply:
Thanks! Changed in [d880fcb](https://github.com/5afe/safe-client-gateway-nest/pull/268/commits/d880fcb3732f5bd60894507bcb3104e4a46b7714)